config/tsconfig.workspace.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "composite": true,
+    "allowJs": true,
+    "checkJs": true
```

Comment:
spotted our tsconfig.base already has `"allowJs": true,` which we could leave out
but i understand might be worth being explicit, since checkJs requires allowJs

Reply:
I think we can actually just eliminate one of these.